ui-frontend/packages/catalog-ui-search/src/main/webapp/component/pages/home.tsx

```diff
@@ -174,6 +174,7 @@ const LeftTop = ({ selectionInterface }: { selectionInterface: any }) => {
           color="primary"
           size="small"
           onClick={() => {
+            selectionInterface.getCurrentQuery().trigger('update')
```

Comment:
This surprises me a bit, I think maybe we need to update query-basic similarly to query-advanced and have it directly modify / use the filter tree as the single source of truth.

Reply:
Yeah, I agree- I'm hoping to update it 👍